### PR TITLE
basic structure for hackathon

### LINKED ISSUE #3: create section for hackathon
folder + README for hackathon

Notes https://docs.google.com/document/d/14D1wpiytpJ4CYrLNz33lWEw-wNL6Ny83jewP3BJY1Bo/edit

### PR DESCRIPTION
**Pull request by orbeckst**
- added Content table to toplevel README
- added hackathon subdir with README
- added mdakits and blender under hackathon with stub-READMEs
- close #3

### CODE REVIEW

**Comment by orbeckst** [comment]
@yuxuanzhuang you're doing the blender hackathon, right? I started adding a stub for `blender/README.md` but you'll have to write it properly. Eg write what your plan is.


Start adding materials for the hackathon in the hackathon/blender subdirectory once the PR is merged.

**Comment by yuxuanzhuang** [comment]
No problem!

File: hackathon/mdakits/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Build an MDAKit
+
+[MDAKits](https://mdakits.mdanalysis.org/) are
+standalone packages containing code using MDAnalysis components that
+solves a specific scientific problems or in some form enhances the
```
Comment (fiona-naughton) [inline hackathon/mdakits/README.md]:
```suggestion
solve a specific scientific problems or in some form enhances the
```

File: hackathon/mdakits/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## Team
+
+The MDAKits team (@ianmkenney, @lilyminium, @fiona-naughton, @IAlibay,
+@orbeckst) will work with you on 
+
+
+## About MDAKits...
+
+A MDAKit can be registered in the [MDAKits
+registry](https://mdakits.mdanalysis.org/mdakits.html) and ar thus be
```
Comment (fiona-naughton) [inline hackathon/mdakits/README.md]:
```suggestion
registry](https://mdakits.mdanalysis.org/mdakits.html) and thus be
```

File: hackathon/mdakits/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+[MDAKits](https://mdakits.mdanalysis.org/) are
+standalone packages containing code using MDAnalysis components that
+solves a specific scientific problems or in some form enhances the
+functionality of MDAnalysis core library. An MDAKit can be written by
+anyone and hosted anywhere.
+
+## Team
+
+The MDAKits team (@ianmkenney, @lilyminium, @fiona-naughton, @IAlibay,
+@orbeckst) will work with you on
```
Comment (fiona-naughton) [inline hackathon/mdakits/README.md]:
```suggestion
@orbeckst) will work with you on creating an MDAKit.
```

**Review by fiona-naughton — APPROVED** [review]
lgtm - thanks @orbeckst !

**Pull request merged**